### DISCUSSION TITLE
Added: LDM notes for January 26th, 2022

### DISCUSSION THREAD

**Discussion by 333fred** [dotnet/csharplang#5690] [LDM Notes]
https://github.com/dotnet/csharplang/blob/main/meetings/2022/LDM-2022-01-26.md

## Agenda

1. Open questions in UTF-8 string literals

**Comment by TahirAhmadov** [comment]
I didn't see any mention of the necessity of doing UTF-8 strings only, instead of tackling the "constant expressions" vertical entirely, which would encompass UTF-8 strings and many other similar uses. I don't find it logical to invent new syntax and go through the entire process to satisfy one niche demand only; what if/when some others start demanding a different type of one-off literal?

**Reply by 333fred** [reply]
UTF-8 strings are becoming increasingly important to the runtime and to our consumers, to an outsized margin. A larger feature would be similarly large in design and implementation cost, and we've made a pragmatic decision to meet the current needs of our customers.
